<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find glove_acquisition)/config/rosconsole_debug_enabled.conf"/>

  <!-- default value to choose whether or not to enable the Glove Publisher verbose mode -->
  <arg name="glove_pub_verbose_mode" default="false"/>

  <node pkg="glove_acquisition" type="glove_talker" name="glove_talker" output="screen" cwd="ROS_HOME">
    <param name="verbose_mode" type="bool" value="$(arg glove_pub_verbose_mode)" />
    <param name="serial_port" type="str" value="/dev/ttyUSB0" />
    <param name="start_comm_char" type="str" value="+" />
    <param name="num_imus" type="int" value="5" />
  </node>
</launch>